build.gradle

```diff
@@ -18,10 +18,11 @@ allprojects {
         options.fork = true  // Fork your compilation into a child process
         options.forkOptions.setMemoryMaximumSize("4g") // Set maximum memory to 4g
         options.compilerArgs << "-Xlint:all"
-        options.compilerArgs << "-Werror"
+        // options.compilerArgs << "-Werror"
```

Comment:
I think I remember correctly, you can exculde this with proguard: 
https://github.com/krschultz/android-proguard-snippets/tree/master/libraries
https://github.com/TCA-Team/TumCampusApp/blob/master/proguard/proguard-project-rules.pro

using the -dontwarn flag